### ISSUE TITLE
No incoming call bubble when the caller is not selected

### ISSUE THREAD

**Issue by eatdust** [qTox/qTox#4989]
Brief Description

OS: Linux Mageia 6
qtox: version 1.13.0
toxcore: 0.1.10
Qt: 5.6.2
Hardware: desktop
…
Reproducible: Always

When I am receiving a call from one of my qtox friend, the ringing sound is heard, but no incoming call bubble is visible.

The only button I can press is the "red phone", which ends up closing the not yet started communication. Then, I can see in the chat window who has tried to call me. If I select the name of that tox friend of mine (by clicking on it in the contact panel), and, if my qtox friend tries to call me again, only now the incoming call bubble appears and I can accept the call.

The issue seems to be that the incoming call bubble is not displayed when the caller is not selected in the contact menu.

Nothing appears in the logs.

Thanks!
Cheers,
Chris.

**Comment by sudden6** [comment]
Are you building qTox yourself and what is the commit hash of your build?

**Comment by eatdust** [comment]
Yes I am, but not only for myself, I am packaging it for Mageia 7 (not yet released).
Therefore, I am downloading only stable releases, so I got the source code from this link:
https://github.com/qTox/qTox/archive/v1.13.0.tar.gz

One thing not standard I maybe using, is a very light window-manager (fvwm2). But I would not expect it to create any bugs, and I have tested it under MATE, same issue.

Compilation flags are however distro-specified:

/usr/bin/cmake .. -DCMAKE_INSTALL_PREFIX:PATH=/usr -DCMAKE_INSTALL_LIBDIR:PATH=/usr/lib64 -DCMAKE_INSTALL_L
IBEXECDIR:PATH=/usr/libexec -DCMAKE_INSTALL_SYSCONFDIR:PATH=/etc -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-DNDEBUG -DCMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-DNDEBUG -DLIB_SUFFIX=64 -DCMAKE_SKIP_RPATH:BOOL=ON -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON '-DCMAKE_MODULE_LINKER_FLAGS=-Wl,--as-needed  -Wl,-z,relro -Wl,-O1 -Wl,--build-id -Wl,--enable-new-dtags' -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF .. '-DCMAKE_CXX_FLAGS=-O2 -g -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -Wno-deprecated-declarations'

**Comment by sudden6** [comment]
If you can wait a few days, the 1.14.0 release is just around the corner. It will also support toxcore 0.2.0. It probably has this fixed and could use some build testing anyway.

**Comment by eatdust** [comment]
Sure, thanks!

**Comment by sudden6** [comment]
release was just tagged

**Comment by sudden6** [comment]
@eatdust Still an issue?

**Comment by dirteat** [comment]
Unfortunately yes, slightly different though!

Now I see an "envelop" nearby of the contact that is trying to start a conversation with me; but if the contact is not selected at the time he is calling, there is no more than the envelop. The incoming call bubble remains hidden.

Thanks!

**Comment by sudden6** [comment]
Thanks for the update, envelope is because we changed the symbol for new messages/activity.
